fotoapparat/src/main/java/io/fotoapparat/hardware/v1/RawCameraParametersProvider.java

```diff
@@ -0,0 +1,82 @@
+package io.fotoapparat.hardware.v1;
+
+import android.hardware.Camera;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Provides {@link Camera.Parameters} with methods to extract additional camera parameters.
+ */
+@SuppressWarnings("deprecation")
+public class RawCameraParametersProvider {
```

Comment:
we use RawCameraParametersProvider with either `getCameraParameters` or `getSensorSensitivityValues` in `CapabilitiesFactory`. I suggest we name this `CameraParametersProvider`, remove the `public getCameraParameters ` and have the previous parameter getters (e.g. `getSupportedPreviewSizes`) as public method of this class which is used in the `CapabilitiesFactory`

Reply:
Agreed. I'll work on it.